core/module/managed_subscribe/src/managed_subscribe_ext.rs

```diff
@@ -148,7 +152,7 @@ impl ManagedSubscribeExt {
     }
 }
 
-impl GrpcExtension for ManagedSubscribeExt {
+impl GrpcModule for ManagedSubscribeExt {
```

Comment:
Rename ManagedSubscribeExt, as we are moving way from using "extension".

Reply:
Updated here and elsewhere